android/uhabits-android/src/main/java/org/isoron/uhabits/notifications/SnoozeDelayPickerActivity.java

```diff
@@ -46,6 +46,14 @@ protected void onCreate(@Nullable Bundle bundle)
         habit = appComponent.getHabitList().getById(parseId(getIntent().getData()));
         if (habit == null) finish();
 
+        // Unlock device on clicking notification action
```

Comment:
Please avoid single line comments explaining what do the next few lines of code do (unless you are in the middle of a complicated algorithm). If you feel that the comment is important enough, please extract the lines of code to a new method and give a descriptive name to the method. In this particular case, we could move it into `unlockScreen()`.